src/main.js

```diff
@@ -14,6 +19,10 @@ var currentCover;
 
 // Add your event listeners here 👇
 showNewRandomCoverButton.addEventListener('click', getRandomCover);
+viewSavedCoversButton.addEventListener('click', function () {
+  currentView = homeView;
```

Comment:
I don't see `currentView` declared anywhere, does it need to be declared first?

Reply:
oh shoot, this was the variable that got removed in your last pr but I thought we added it back!  Fantastic catch, I'll add the declaration back!

Reply:
That's `var currentCover;` lol 
I just watched a video that kinda describes what's happening here and why it's working tho, interesting stuff! [This video.](https://youtu.be/9emXNzqCKyg?t=469)